babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: [["@babel/preset-env", { targets: { node: "current" } }]],
```

Comment:
what is the reason for adding babel? the sdk uses tsup as the bundle tool

Reply:
It is for `babel-jest`

Some imports from `@simplewebauthn/browser` broke jest, and I couldn't find a way to successfully run it using only ts-jest, so I introduced bable-jest.

Reply:
I see. I'd prefer not to introduce another bundle tool and keep the existing one. Also, to reduce potential errors, and to not increase the package size.

Reply:
Agree, but I have spent two days, still cannot run jest with only `ts-jest`, looking forward to any suggestions

Reply:
BTW, the solution I found is from https://github.com/MasterKale/SimpleWebAuthn/issues/293